src/thunks/repositoryThunk.ts

```diff
@@ -20,10 +20,11 @@ export const fetchRecentRevisions = createAsyncThunk<
     let response;
 
     try {
-      response = await fetch(`${treeherderBaseURL}/api/perfcompare/results/?`);
+      response = await fetch(`${treeherderBaseURL}/api/perfcompare/result/?`);
     } catch (err) {
-      const error = (err as Error).message;
-      return rejectWithValue(error);
+      const error = err as Error;
+      console.log('FetchRecentRevisions ERROR: ', error);
```

Comment:
thanks for adding all these console.log!
optional nit: you can use `console.error` for this too, this will stand out with a red background in the devtools console.